index.html

```diff
@@ -0,0 +1,181 @@
+        </p>
+      </section>
+      <section id="Speakers">
+        <h2>
+          Speakers
+        </h2>
+        <h3>Carol Twombly</h3>
+        <img
+          src="https://cdn.glitch.com/45cf91dd-1c76-4720-9ed6-0b9ec5556704%2Fcarol%20twombly.png?v=1608030276977" alt="Carol Twombly picture"
+        />
+        <p>
+          During her eleven+ years with Adobe, Carol has designed a number of
+          very popular text and display typefaces. Designs like Trajan,
+          Charlemagne, Lithos, and Adobe Caslon are inspired by classic
+          letterforms of the past - from early Greek inscriptions, circa 400
+          B.C., to William Caslon’s typefaces of the 1700s. Designs like Viva
+          and Nueva explore new territory while maintaining traditional roots.
+          In 1994, she received the Charles Peignot award from the Association
+          Typographique Internationale for outstanding contributions to type
+          design. She was the first woman and only the second American to
+          receive this prestigious honor.
+        </p>
+        <h3>Mary Kate McDevitt</h3>
+        <img
+          src="https://cdn.glitch.com/45cf91dd-1c76-4720-9ed6-0b9ec5556704%2Fmary%20kate%20mcdevitt.jpg?v=1608030309268" alt="Mary Kate Mcdevitt picture"
+        />
+        <p>
+          A letterer and illustrator, McDevitt lives and works in Brooklyn, New
+          York. Some of her prominent clients include AT&T, Penguin Books, Nike,
+          United States Postal Service, Sesame Street, and Target. She also
+          worked with Chronicle Books, Nintendo, and Wall Street Journal, as
+          well as Smart Ones, Fast Company and O Magazine. McDevitt had a degree
+          in Graphic Design and Illustration from Tyler School of Art in
+          Philadelphia.
+        </p>
+        <h3>Marian Bantjes</h3>
+        <img
+          src="https://cdn.glitch.com/45cf91dd-1c76-4720-9ed6-0b9ec5556704%2Fmarian%20bantjes.jpg?v=1608030384123" alt="Marian Bantjes"
+        />
+        <p>
+          Having worked as a book typesetter and founding the Digitopolis
+          graphic design studio, Canadian designer Marian Bantjes currently
+          works on her own projects in design, art, and typography. Her
+          ornamental, often dreamlike style has been featured in over 100 books,
+          including her own beautiful I Wonder in 2010 (with a foreword by
+          Stefan Sagmeister of the aforementioned Sagmeister & Walsh). Bantjes
+          continues to bring new and unusual letterforms into the design
+          vocabulary.
+        </p>
+      </section>
+      <section id="Register">
+        <h2>
+          Register
+        </h2>
+        <p class="important-text">
+          <em>$3.00</em> -Conference entering fee<br />
+          <em>$5.00</em> -Selling Booth registration fee<br />
+          click the button for the conference registration!<br />
+        </p>
+        <button>
+          Register
+        </button>
```

Comment:
It's unclear to me what happens when a user clicks this button -- if they are sent off to a different page or site to register for the conference, this should instead be an anchor element, like so:

```suggestion
        <a href="https://eventbrite.com">Register</a>
```